sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/StorageApiSinkRowUpdateIT.java

```diff
@@ -130,16 +101,19 @@ public void testCdc() throws Exception {
                 new TableRow().set("key1", "foo4").set("key2", "bar4").set("value", "1"),
                 RowMutationInformation.of(RowMutationInformation.MutationType.DELETE, 1)));
 
-    String tableSpec = createTable(tableSchema, Lists.newArrayList("key1", "key2"));
+    List<String> primaryKey = Lists.newArrayList("key1", "key2");
+    String tableSpec = getTablespec();
     Pipeline p = Pipeline.create();
     p.apply("Create rows", Create.of(items))
         .apply(
             "Apply updates",
             BigQueryIO.applyRowMutations()
                 .to(tableSpec)
                 .withSchema(tableSchema)
+                .withPrimaryKey(primaryKey)
+                .withClustering(new Clustering().setFields(primaryKey))
```

Comment:
nit: is there a need to set clustering in this test case?

Reply:
It was at the time I wrote the test, though I think BQ plans to remove this restriction. I'm not sure if it's been removed yet.

Reply:
I pulled these changes locally and ran the test without clustering, the test passes fine

Reply:
If clustering is indeed needed though, it would be good to update the use cases in BigQueryIO.java doc comments

Reply:
it sounds like it might be safer to leave clustering here for now - I think BigQuery is in the process of removing this restriction, but it might not be completely removed yet.

Reply:
Looks like BigQuery is removing the requirement for clustering but it's not rolled out everywhere yet, let's keep the test as is and no need to update docs